app/core/ViewVisibleTimer.coffee

```diff
@@ -61,7 +61,9 @@ class ViewVisibleTimer extends CocoClass
       @running = true
       @startTime = window.performance.now()
 
-  stopTimer: ({ subtractTimeout = false, clearName = false } = { })->
+  stopTimer: ({ subtractTimeout, clearName } = { })->
```

Comment:
👍
FYI our version of CoffeeScript now doesn't support this syntax for some reason.